src/lib/data/near.js

```diff
@@ -356,3 +356,14 @@ export const useNear = singletonHook(defaultNear, () => {
 
   return near;
 });
+
+export const useMainnetNear = () => {
+  const [near, setNear] = useState(defaultNear);
+  const nearPromise = useMemo(() => _initNear({networkId: 'mainnet'}), []);
```

Comment:
I don't think default config is a good way moving forward. It doesn't have `customElements`, custom API server endpoint and also no wallet selector.

Better to introduce multi network config to the `initNear` and use one as a default.

Then support `<Widget chainId="mainnet" ...>`

Reply:
Looking into it but we will need to use context here since the `chainId` value will need to be set for `Widget` components that result from fetching the src of the parent `Widget` component. Unless there is a way to drill this prop down to resulting children in the VM compilation.

Reply:
Ah, good point. You can utilize `config` attribute on the `Widget` to pass `chainId`

Reply:
Thanks. I got it to work with context by accessing the config in the first index of the `configs` state